util/hb-info.cc

```diff
@@ -190,12 +194,37 @@ struct info_t
     printf ("%u\n", hb_face_get_glyph_count (face));
   }
 
+  void _list_names ()
+  {
+    if (verbose)
+    {
+      separator ();
+      printf ("Name information:\n\n");
+      printf ("Id	Text\n\n");
+    }
+
+    auto language = hb_language_get_default ();
+
+    unsigned count;
+    const auto *entries = hb_ot_name_list_names (face, &count);
+    for (unsigned i = 0; i < count; i++)
+    {
+      char name[128];
+      unsigned name_len = sizeof name;
+      hb_ot_name_get_utf8 (face, entries[i].name_id,
+			   language,
+			   &name_len, name);
+
+      printf ("%u	%s\n", entries[i].name_id, name);
```

Comment:
It would be nice to separate-out the pre-defined name IDs and add a human name next to the ID.

Reply:
That's too much work. I'm adding separate `--show-family` etc, which will be in the default output.

Reply:
Same thing with feature names, script names, etc. We don't have that information accessible.

Reply:
It can be something like (assuming this is valid C code 😄 :
```C
char* pre_defined_names[] = {
  "Copyright",
  "Family",
   ...
};

if (entries[i].name_id < sizeof (pre_defined_names))
  printf ("%s (%u):	%s\n", entries[i].name_id, name);
else
  printf ("%u	%s\n", entries[i].name_id, name);
```

Reply:
> Same thing with feature names, script names, etc. We don't have that information accessible.

This list is shorter and tags are memorable unlike numeric IDs, but I won’t insist.